airbyte-integrations/bases/base-normalization/normalization/transform_catalog/destination_name_transformer.py

```diff
@@ -29,6 +29,8 @@
     DestinationType.CLICKHOUSE.value: 63,
     # https://docs.pingcap.com/tidb/stable/tidb-limitations
     DestinationType.TIDB.value: 64,
+    # Couldn't find
```

Comment:
I found the `character_maximum_length` on https://duckdb.org/docs/sql/information_schema.html. But I'm guessing that is the content and not the column name length. I asked in their Discord to clarify. Otherwise I will add a more specific comment that I'm not sure.

Also, when I had it 1024, I got an error. When I reduced to 64 works, that's why I added a defenisive number that I know worked. I will follow up.

Reply:
According to the DuckDB team: We don't enforce a maximum right now but I would not recommend having column names longer than a few kilobytes. I added the link and updated to 255 max, as I had problems with 1024. But can always raised later.